examples/light.py

```diff
@@ -0,0 +1,60 @@
+import time
```

Comment:
A short docstring to explain what these new examples demonstrate would be nice :)